config/locales/idv/fr.yml

```
@@ -145,12 +145,18 @@ fr:
         information vérifiée, veuillez %{link}.
       activated_link: communiquer avec nous
       clear_and_start_over: Supprimer mes données et recommencer
-      come_back_later: Une fois que votre lettre vous sera parvenue, veuillez vous
-        connecter à %{app_name}, puis saisissez votre code à usage unique
-        lorsque vous y serez invité.
+      come_back_later_html: <p><strong>Les lettres prennent généralement trois à sept
+        jours ouvrables pour arriver</strong>< br/> par le biais du courriel de
```

Not sure if it would cause a problem or be considered invalid markup, but there's a misplaced space on the line break element here.

```suggestion
        jours ouvrables pour arriver</strong><br /> par le biais du courriel de
```